### PR TITLE
add project: soulswap

### PR DESCRIPTION
**Pull request by BunsDev**
##### Twitter Link:
https://twitter.com/SoulSwapFinance

##### List of audit links if any:
N/A

##### Website Link:
https://soul.sh

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://raw.githubusercontent.com/SoulSwapFinance/assets/prod/soul-200x200.png

##### Current TVL:
89.45K USD

##### Chain:
Fantom Opera (ChainID: 250)

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
TBD - Pending Application

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
TBD - Pending Application

##### Short Description (to be shown on DefiLlama):
Soul is an algorithmic, cross-chain AMM and P2P lending protocol (on Fantom) built for traders, investors, developers, and visionaries seeking to unlock a universe of open financial applications.

##### Token address and ticker if any:
SOUL (SoulPower)
0xe2fb177009ff39f52c0134e8007fa0e4baacbd07
https://ftmscan.com/token/0xe2fb177009ff39f52c0134e8007fa0e4baacbd07

##### Category (Yield/Dexes/Lending/Minting/Assets/Insurance/Options/Indexes/Staking) *Please choose only one:
Dexes

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
N/A

##### forkedFrom (Does your project originate from another project):
SushiSwap, PancakeSwap, Uniswap, Synthetix, Yam

### CODE REVIEW

File: projects/soulswap/helper/getBlock.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const sdk = require('@defillama/sdk')
+const retry = require('async-retry')
+const axios = require('axios');
+
+async function getBlock(timestamp, chain, chainBlocks) {
+    if(chain === 'celo'){
```
Comment (0xngmi) [inline projects/soulswap/helper/getBlock.js:6]: why fork this file?

**Comment by 0xngmi** [comment]
some of the files you forked don't seem necessary, could you remove them please?

File: projects/soulswap/helper/getBlock.js (hunk 1 above)
Comment (BunsDev) [reply projects/soulswap/helper/getBlock.js]: thanks for letting me know. I have now deleted redundant files and updated paths to pull from helper global import.

**Comment by BunsDev** [comment]
fixed the conflict with the missing package-lock.json @0xngmi

**Pull request merged**